editoast/src/views/rolling_stock.rs

```diff
@@ -1,5 +1,6 @@
 pub mod form;
 pub mod light;
+pub mod towed;
```

Comment:
```suggestion
mod towed;
```

I think this doesn't need to be `pub`, could be totally wrong though.

Reply:
You're right, done in 54d9f7c3ae23bc027ae0d56b39ed1a83dd4d65f8.